bazel/grpc_python_deps.bzl

```diff
@@ -37,9 +37,9 @@ def grpc_python_deps():
     http_archive(
```

Comment:
nit: let's add a comment that this should be synced with the internal version, similar to this:
https://github.com/grpc/grpc-java/blob/42e1829b3724c0fb20910c0abe70099994856307/xds/third_party/xds/import.sh#L19

> `# import VERSION from one of the google internal CLs`